<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>321</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>접속중:</string>
   </property>
  </widget>
  <widget class="QLabel" name="peopleNum">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="userNameLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>67</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>내 이름</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
